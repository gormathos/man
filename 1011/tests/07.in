sol_id;user_id;task_id;score
112382;1016;2005;76.92
112392;1015;2004;28.57
112393;1028;2005;0.00
112399;1020;2001;33.33
112404;1001;2002;34.38
112407;1025;2002;31.25
112412;1025;2003;48.65
112417;1026;2005;42.31
112425;1020;2002;3.12
112434;1006;2005;53.85
112437;1001;2001;21.43
112444;1013;2004;83.33
112449;1027;2002;59.38
112456;1030;2003;64.86
112460;1020;2003;37.84
112467;1006;2002;46.88
112468;1011;2004;14.29
112476;1003;2002;68.75
112480;1010;2001;23.81
112485;1030;2005;0.00
112492;1016;2001;54.76
112492;1013;2003;8.11
112498;1005;2001;14.29
112500;1018;2005;84.62
112506;1027;2003;8.11
112516;1020;2002;0.00
112521;1027;2001;19.05
112523;1010;2002;0.00
112526;1017;2005;50.00
112533;1025;2002;21.88
112542;1024;2003;35.14
112551;1019;2004;16.67
112553;1005;2005;42.31
112557;1003;2002;34.38
112560;1013;2004;59.52
112562;1009;2003;78.38
112571;1005;2005;100.00
112575;1012;2005;42.31
112579;1017;2005;50.00
112585;1009;2001;28.57
112595;1026;2002;9.38
112597;1016;2003;24.32
112599;1009;2002;62.50
112609;1026;2001;7.14
112612;1019;2001;35.71
112621;1011;2001;16.67
112629;1003;2001;64.29
112632;1009;2002;15.62
112635;1005;2001;35.71
112644;1020;2005;100.00
112645;1011;2004;30.95
112646;1023;2001;76.19
112653;1007;2001;14.29
112656;1018;2002;96.88
112662;1006;2003;18.92
112666;1016;2004;14.29
112670;1003;2002;3.12
112676;1003;2001;54.76
112682;1021;2003;54.05
112690;1028;2003;78.38
112698;1030;2004;50.00
112698;1021;2004;7.14
112706;1026;2005;88.46
112708;1030;2004;26.19
112714;1002;2001;33.33
112717;1018;2005;65.38
112722;1020;2004;71.43
112725;1021;2004;4.76
112728;1018;2004;76.19
112732;1011;2003;18.92
112738;1008;2001;4.76
112741;1028;2002;46.88
112749;1020;2001;47.62
112756;1010;2003;8.11
112765;1005;2005;76.92
112768;1012;2001;11.90
112771;1025;2002;90.62
112777;1029;2002;65.62
112777;1009;2003;27.03
112785;1021;2004;11.90
112788;1024;2002;43.75
112788;1008;2002;56.25
112794;1022;2005;38.46
112795;1003;2005;50.00
112800;1021;2002;0.00
112802;1006;2005;19.23
112803;1004;2002;65.62
112806;1028;2005;80.77
112806;1014;2003;5.41
112810;1016;2003;21.62
112818;1016;2001;2.38
112822;1014;2003;29.73
112829;1014;2004;61.90
112837;1013;2001;95.24
112841;1008;2003;0.00
112845;1025;2001;0.00
112846;1025;2003;0.00
112848;1018;2001;33.33
112853;1023;2003;100.00
112855;1019;2004;23.81
112858;1020;2002;12.50
112861;1012;2003;83.78
112868;1009;2002;6.25
112872;1026;2003;8.11
112874;1007;2004;61.90
112876;1027;2005;19.23
112880;1027;2004;4.76
112883;1016;2005;15.38
112884;1001;2005;88.46
112889;1027;2003;0.00
112893;1021;2001;35.71
112902;1013;2005;19.23
112902;1021;2005;23.08
112904;1007;2004;4.76
112904;1018;2005;69.23
112913;1010;2004;11.90
112918;1019;2003;29.73
112919;1021;2004;7.14
112921;1026;2001;26.19
112929;1019;2001;52.38
112938;1029;2001;2.38
112944;1030;2001;7.14
112949;1026;2003;18.92
112952;1009;2003;32.43
112960;1026;2003;13.51
112963;1002;2001;50.00
112971;1022;2005;96.15
112979;1019;2002;78.12
112984;1006;2003;51.35
112994;1024;2002;93.75
113004;1008;2004;14.29
113009;1014;2002;21.88
113019;1015;2003;67.57
113022;1024;2003;10.81
113024;1027;2002;6.25
113034;1029;2003;0.00
113036;1028;2003;16.22
113045;1024;2003;21.62
113055;1016;2003;67.57
113056;1011;2001;38.10
113058;1018;2002;12.50
113063;1003;2001;30.95
113072;1012;2003;24.32
113081;1029;2003;2.70
113091;1020;2002;62.50
113099;1023;2001;85.71
113105;1026;2001;66.67
113107;1028;2004;16.67
113115;1002;2001;50.00
113124;1003;2002;15.62
113130;1015;2001;28.57
113133;1013;2003;81.08
113135;1015;2004;11.90
113144;1005;2004;28.57
113152;1028;2004;23.81
113156;1002;2004;71.43
113158;1007;2002;84.38
113159;1002;2005;34.62
113161;1030;2003;67.57
113170;1004;2001;76.19
113180;1019;2004;35.71
113184;1018;2003;59.46
113192;1001;2001;19.05
113197;1022;2005;0.00
113204;1022;2003;0.00
113207;1003;2003;5.41
113210;1003;2001;4.76
113217;1007;2004;19.05
113221;1019;2003;24.32
113222;1029;2002;50.00
113229;1009;2002;34.38
113236;1016;2002;56.25
113241;1025;2004;0.00
113250;1006;2001;2.38
113253;1014;2004;0.00
113262;1013;2005;3.85
113266;1020;2003;45.95
113273;1027;2003;16.22
113281;1001;2005;23.08
113287;1013;2005;30.77
113292;1019;2003;29.73
113295;1001;2001;80.95
113302;1029;2005;69.23
113305;1010;2001;66.67
113315;1019;2004;38.10
113317;1013;2003;45.95
113326;1002;2004;0.00
113333;1029;2005;23.08
113339;1029;2001;71.43
113348;1018;2003;29.73
113352;1009;2005;15.38
113357;1013;2002;28.12
113359;1015;2005;61.54
113360;1020;2004;54.76
113367;1013;2004;57.14
113372;1027;2001;35.71
113373;1025;2005;65.38
113377;1014;2003;5.41
113379;1014;2004;21.43
113384;1020;2003;0.00
113393;1003;2002;0.00
113401;1005;2005;15.38
113411;1001;2004;2.38
113417;1016;2004;19.05
113420;1027;2004;9.52
113423;1003;2005;11.54
113429;1029;2003;64.86
113439;1025;2002;53.12
113441;1009;2003;32.43
113449;1006;2004;78.57
113456;1018;2004;78.57
113461;1025;2003;24.32
113466;1004;2003;13.51
113467;1012;2002;34.38
113473;1015;2002;53.12
113473;1019;2001;92.86
113481;1006;2003;21.62
113491;1012;2003;18.92
113496;1026;2004;45.24
113497;1023;2004;59.52
113507;1017;2004;9.52
113516;1019;2002;68.75
113519;1016;2004;19.05
113522;1002;2003;59.46
113524;1022;2002;0.00
113526;1025;2005;19.23
113536;1020;2001;23.81
113538;1013;2004;11.90
113544;1018;2004;38.10
113550;1030;2002;84.38
113558;1020;2004;47.62
113566;1006;2002;0.00
113572;1016;2002;78.12
113579;1012;2004;23.81
113580;1029;2005;3.85
113587;1023;2002;65.62
113596;1012;2002;71.88
113602;1015;2005;34.62
113604;1023;2004;47.62
113604;1023;2005;42.31
113608;1007;2001;26.19
113615;1004;2002;12.50
113623;1003;2003;0.00
113633;1001;2004;23.81
113633;1014;2005;7.69
113642;1014;2001;0.00
113648;1019;2002;25.00
113648;1003;2004;83.33
113656;1002;2002;9.38
113658;1004;2004;30.95
113658;1001;2005;15.38
113665;1027;2001;33.33
113668;1015;2001;64.29
113676;1012;2005;53.85
113679;1010;2004;26.19
113679;1007;2003;94.59
113689;1030;2004;14.29
113693;1030;2003;0.00
113703;1017;2001;0.00
113711;1001;2003;8.11
113717;1010;2005;19.23
113720;1015;2004;21.43
113723;1003;2003;8.11
113731;1002;2004;2.38
113741;1005;2004;33.33
113750;1014;2003;27.03
113758;1015;2004;16.67
113764;1007;2001;11.90
113769;1002;2005;69.23
113779;1028;2005;0.00
113783;1007;2004;40.48
113789;1022;2004;38.10
113790;1013;2003;97.30
113800;1027;2004;23.81
113810;1003;2003;5.41
113817;1002;2005;3.85
113824;1024;2002;0.00
113824;1012;2001;30.95
113830;1030;2004;0.00
113839;1028;2005;11.54
113849;1006;2002;28.12
113849;1013;2005;3.85
113850;1008;2001;2.38
113856;1014;2001;30.95
113866;1017;2002;0.00
113873;1017;2005;61.54
113880;1003;2004;42.86
113881;1023;2004;61.90
113882;1027;2001;28.57
113890;1001;2001;0.00
113891;1011;2002;6.25
113899;1029;2002;46.88
113902;1011;2005;3.85
113905;1024;2005;0.00
113914;1026;2002;15.62
113922;1021;2003;0.00
113924;1016;2001;0.00
113928;1025;2005;57.69
113933;1026;2001;4.76
113935;1021;2002;43.75
113942;1022;2003;37.84
113950;1008;2004;50.00
113954;1019;2003;100.00
113961;1005;2004;16.67
113967;1009;2004;45.24
113967;1026;2005;3.85
113970;1005;2004;35.71
113980;1024;2004;14.29
113984;1006;2004;2.38
113990;1004;2003;13.51
113996;1001;2004;19.05
114003;1018;2005;0.00
114004;1001;2003;48.65
114011;1021;2003;2.70
114017;1026;2002;34.38
114026;1002;2005;65.38
114030;1019;2005;34.62
114039;1008;2001;40.48
114047;1018;2001;69.05
114050;1028;2002;75.00
114052;1017;2004;85.71
114060;1007;2002;6.25
114068;1006;2004;64.29
114070;1018;2004;40.48
114078;1005;2004;40.48
114086;1007;2005;15.38
114096;1006;2004;9.52
114102;1022;2005;0.00
114105;1029;2001;21.43
114115;1022;2002;12.50
114122;1001;2004;71.43
114124;1023;2005;19.23
114127;1025;2002;3.12
114134;1011;2001;4.76
114143;1005;2003;8.11
114151;1017;2005;3.85
114152;1008;2001;42.86
114155;1018;2002;12.50
114160;1004;2001;73.81
114165;1008;2005;19.23
114166;1030;2001;4.76
114166;1002;2001;40.48
114169;1004;2002;31.25
114173;1014;2004;42.86
114177;1026;2001;7.14
114187;1022;2003;37.84
114193;1010;2001;4.76
114196;1001;2003;27.03
114197;1008;2003;0.00
114198;1016;2002;59.38
114206;1029;2004;80.95
114206;1018;2003;37.84
114207;1019;2005;0.00
114212;1007;2001;33.33
114213;1010;2004;100.00
114220;1018;2001;23.81
114230;1004;2004;4.76
114237;1004;2001;7.14
114247;1018;2002;43.75
114248;1019;2002;84.38
114256;1022;2003;62.16
114264;1023;2002;21.88
114271;1014;2005;53.85
114273;1017;2003;10.81
114274;1023;2001;28.57
114280;1023;2004;28.57
114284;1028;2001;2.38
114291;1016;2005;92.31
114297;1024;2001;97.62
114304;1003;2004;54.76
114314;1030;2003;24.32
114319;1024;2005;19.23
114325;1021;2003;86.49
114328;1015;2003;94.59
114329;1029;2004;30.95
114336;1016;2004;9.52
114344;1013;2002;6.25
114347;1022;2001;52.38
114348;1008;2005;50.00
114348;1020;2005;65.38
114350;1019;2002;25.00
114353;1006;2003;0.00
114358;1002;2002;9.38
114360;1023;2001;30.95
114370;1022;2001;38.10
114378;1022;2003;24.32
114378;1009;2003;97.30
114386;1025;2003;13.51
114393;1011;2004;38.10
114399;1008;2001;97.62
114409;1024;2001;2.38
114418;1019;2003;32.43
114427;1014;2002;84.38
114436;1012;2004;47.62
114440;1009;2004;26.19
114443;1028;2002;53.12
114447;1017;2002;31.25
114449;1014;2002;3.12
114456;1013;2001;88.10
114459;1001;2004;9.52
114466;1013;2001;40.48
114468;1021;2002;46.88
114476;1009;2004;35.71
114478;1014;2005;0.00
114483;1013;2003;54.05
114485;1009;2003;86.49
114493;1015;2003;81.08
114500;1018;2001;80.95
114509;1014;2005;69.23
114516;1004;2004;54.76
114523;1020;2001;23.81
114533;1008;2002;46.88
114543;1016;2001;33.33
114550;1002;2002;6.25
114555;1016;2001;35.71
114561;1011;2005;23.08
114563;1024;2002;100.00
114571;1008;2002;28.12
114571;1016;2001;4.76
114578;1024;2001;14.29
114580;1006;2001;73.81
114587;1023;2004;78.57
114597;1014;2001;69.05
114598;1004;2003;0.00
114601;1013;2001;97.62
114607;1015;2005;3.85
114616;1025;2003;18.92
114616;1011;2002;84.38
114619;1007;2002;71.88
114623;1007;2005;3.85
114624;1007;2004;14.29
114625;1017;2001;42.86
114633;1013;2004;35.71
114643;1003;2003;2.70
114649;1004;2004;2.38
114650;1017;2002;93.75
114660;1026;2003;16.22
114661;1025;2001;16.67
114661;1010;2002;37.50
114666;1005;2001;40.48
114668;1006;2005;3.85
114672;1012;2004;95.24
114677;1017;2003;83.78
114686;1005;2002;18.75
114696;1006;2001;88.10
114702;1020;2003;13.51
114703;1013;2001;30.95
114713;1019;2001;59.52
114719;1020;2002;9.38
114728;1020;2001;23.81
114732;1008;2002;50.00
114739;1015;2001;40.48
114745;1026;2004;11.90
114749;1022;2005;53.85
114751;1028;2004;59.52
114753;1008;2001;50.00
114758;1014;2002;6.25
114760;1015;2004;9.52
114770;1026;2003;27.03
114772;1028;2001;23.81
114772;1019;2001;73.81
114778;1021;2001;90.48
114782;1016;2004;97.62
114787;1003;2001;16.67
114795;1012;2002;6.25
114798;1009;2004;28.57
114800;1023;2001;97.62
114804;1018;2004;92.86
114813;1025;2005;23.08
114821;1024;2004;61.90
114824;1004;2005;11.54
114834;1005;2002;12.50
114843;1023;2003;64.86
114843;1002;2001;19.05
114848;1026;2004;21.43
114851;1005;2005;100.00
114854;1017;2003;2.70
114856;1003;2004;2.38
114857;1030;2002;28.12
114866;1009;2004;14.29
114869;1002;2004;78.57
114873;1008;2003;83.78
114879;1006;2005;19.23
114884;1021;2005;30.77
114886;1030;2005;23.08
114894;1018;2002;31.25
114900;1011;2004;23.81
114901;1012;2003;94.59
114908;1010;2005;11.54
114915;1008;2005;61.54
114921;1022;2001;21.43
114926;1004;2005;7.69
114927;1025;2001;9.52
114928;1009;2005;76.92
114932;1008;2002;78.12
114934;1029;2002;68.75
114942;1007;2005;23.08
114949;1013;2001;19.05
114958;1015;2005;30.77
114966;1021;2005;65.38
114970;1025;2004;35.71
114970;1016;2002;21.88
114977;1017;2005;61.54
114987;1023;2003;45.95
114990;1016;2005;19.23
114999;1019;2004;59.52
115003;1005;2003;89.19
115013;1022;2002;18.75
115020;1004;2005;19.23
115020;1002;2003;29.73
115030;1010;2003;59.46
115030;1002;2002;50.00
115036;1022;2004;11.90
115046;1023;2003;24.32
115049;1028;2001;14.29
115053;1023;2005;26.92
115060;1015;2003;13.51
115060;1026;2004;50.00
115067;1009;2001;35.71
115076;1020;2004;0.00
115082;1002;2004;52.38
115092;1001;2005;19.23
115092;1029;2004;69.05
115098;1030;2005;61.54
115105;1009;2004;30.95
115115;1014;2003;51.35
115124;1018;2005;3.85
115130;1007;2003;10.81
115134;1024;2001;28.57
115141;1005;2002;37.50
115144;1020;2002;25.00
115152;1029;2002;96.88
115160;1012;2002;75.00
115163;1001;2003;8.11
115170;1006;2005;11.54
115179;1021;2005;30.77
115189;1025;2004;16.67
115199;1019;2005;42.31
115201;1025;2004;21.43
115207;1010;2005;46.15
115217;1023;2005;53.85
115217;1012;2004;16.67
115226;1007;2005;26.92
115234;1010;2004;21.43
115238;1028;2004;76.19
115246;1003;2001;100.00
115255;1010;2004;14.29
115265;1027;2001;26.19
115275;1028;2003;21.62
115285;1028;2005;34.62
115287;1010;2005;0.00
115290;1005;2005;3.85
115294;1010;2001;42.86
115297;1007;2005;50.00
115302;1028;2005;73.08
115312;1007;2004;2.38
115312;1011;2003;13.51
115316;1006;2005;11.54
115325;1011;2003;40.54
115335;1002;2005;3.85
115335;1008;2005;50.00
115338;1010;2004;85.71
115346;1011;2003;8.11
115356;1023;2005;57.69
115360;1020;2003;2.70
115363;1009;2005;96.15
115366;1026;2004;71.43
115372;1009;2001;57.14
115380;1008;2002;15.62
115382;1024;2005;26.92
115385;1024;2004;47.62
115388;1027;2004;45.24
115391;1004;2005;7.69
115391;1012;2002;21.88
115393;1024;2002;37.50
115399;1013;2002;71.88
115405;1022;2002;81.25
115405;1011;2001;59.52
115408;1029;2001;26.19
115410;1022;2002;37.50
115420;1003;2004;52.38
115429;1014;2002;37.50
115434;1019;2005;34.62
115442;1015;2003;64.86
115443;1023;2004;2.38
115450;1003;2005;38.46
115452;1026;2005;57.69
115455;1023;2001;42.86
115456;1010;2002;100.00
115463;1006;2004;14.29
115466;1013;2005;15.38
115472;1022;2001;88.10
115474;1021;2001;73.81
115476;1006;2001;50.00
115483;1011;2003;86.49
115483;1025;2002;3.12
115491;1030;2001;2.38
115498;1001;2001;40.48
115498;1025;2004;45.24
115498;1004;2002;0.00
115502;1010;2003;35.14
115508;1010;2002;43.75
115512;1007;2003;24.32
115517;1023;2002;59.38
115520;1030;2002;37.50
115527;1027;2005;26.92
115527;1030;2004;0.00
115527;1009;2005;65.38
115533;1010;2002;0.00
115539;1022;2004;26.19
115539;1006;2002;3.12
115541;1023;2002;12.50
115546;1011;2002;28.12
115549;1016;2003;89.19
115557;1011;2002;9.38
115563;1006;2002;0.00
115565;1025;2001;97.62
115572;1002;2004;42.86
115572;1013;2005;65.38
115575;1007;2005;15.38
115582;1024;2004;80.95
115589;1029;2005;0.00
115594;1029;2003;29.73
115600;1017;2003;18.92
115601;1029;2004;2.38
115605;1005;2003;8.11
115612;1019;2002;3.12
115618;1004;2001;23.81
115623;1029;2003;27.03
115626;1020;2004;4.76
115633;1004;2002;31.25
115634;1027;2004;100.00
115642;1025;2005;73.08
115650;1014;2002;21.88
115659;1003;2005;53.85
115660;1005;2003;35.14
115662;1016;2002;0.00
115664;1001;2004;80.95
115664;1012;2001;40.48
115670;1027;2004;71.43
115671;1002;2005;7.69
115674;1008;2005;69.23
115675;1008;2004;28.57
115683;1015;2003;10.81
115688;1022;2005;80.77
115690;1030;2001;16.67
115694;1009;2001;21.43
115699;1008;2004;2.38
115701;1023;2002;18.75
115703;1002;2002;0.00
115705;1012;2005;53.85
115706;1029;2005;0.00
115716;1001;2003;62.16
115724;1021;2003;62.16
115733;1028;2002;100.00
115734;1014;2004;38.10
115738;1022;2002;0.00
115740;1029;2001;2.38
115749;1021;2001;100.00
115749;1025;2001;7.14
115758;1019;2004;54.76
115758;1022;2001;42.86
115763;1015;2002;75.00
115763;1030;2005;42.31
115765;1001;2002;37.50
115767;1022;2004;11.90
115767;1024;2003;29.73
115776;1001;2002;62.50
115779;1026;2004;45.24
115785;1011;2004;47.62
115791;1030;2001;97.62
115793;1001;2002;0.00
115796;1021;2003;16.22
115798;1021;2002;21.88
115800;1028;2003;51.35
115807;1017;2001;71.43
115815;1004;2003;64.86
115819;1014;2005;34.62
115822;1004;2002;0.00
115828;1014;2001;0.00
115830;1024;2001;26.19
115833;1016;2005;57.69
115841;1029;2002;12.50
115844;1015;2002;3.12
115847;1004;2003;16.22
115853;1010;2003;13.51
115858;1018;2003;0.00
115861;1011;2002;78.12
115863;1027;2002;15.62
115870;1029;2005;50.00
115878;1019;2001;0.00
115879;1015;2001;42.86
115880;1007;2001;7.14
115888;1012;2003;8.11
115894;1018;2001;14.29
115900;1007;2001;30.95
115909;1029;2001;85.71
115917;1018;2002;12.50
115919;1005;2004;69.05
115924;1024;2005;38.46
115927;1027;2005;88.46
115928;1003;2004;30.95
115930;1014;2001;50.00
115937;1004;2001;100.00
115942;1007;2003;18.92
115951;1012;2004;26.19
115954;1030;2002;12.50
115962;1029;2004;2.38
115972;1024;2004;19.05
115982;1021;2004;69.05
115992;1018;2003;72.97
115994;1020;2005;34.62
115994;1030;2005;3.85
115994;1004;2005;7.69
116002;1001;2001;14.29
116008;1019;2005;57.69
116010;1026;2002;9.38
116013;1005;2003;18.92
116016;1021;2002;6.25
116016;1007;2002;3.12
116025;1001;2005;0.00
116029;1022;2004;64.29
116030;1012;2001;11.90
116035;1012;2005;73.08
116036;1024;2004;90.48
116036;1012;2004;88.10
116042;1001;2005;73.08
116052;1009;2002;9.38
116059;1016;2002;53.12
116067;1028;2001;45.24
116071;1017;2004;7.14
116073;1024;2002;21.88
116081;1003;2005;50.00
116091;1025;2003;32.43
116093;1026;2003;2.70
116094;1030;2003;89.19
116098;1027;2005;15.38
116104;1006;2004;7.14
116105;1001;2002;37.50
116111;1023;2002;93.75
116114;1005;2002;46.88
116118;1027;2005;80.77
116125;1015;2001;7.14
116131;1017;2001;47.62
116136;1015;2004;4.76
116145;1024;2003;21.62
116148;1024;2005;26.92
116151;1013;2004;2.38
116152;1011;2003;18.92
116155;1022;2004;4.76
116159;1026;2001;0.00
116163;1020;2005;23.08
116164;1029;2004;95.24
116174;1025;2001;21.43
116175;1002;2002;6.25
116178;1028;2003;72.97
116184;1004;2003;72.97
116185;1016;2004;71.43
116194;1012;2001;14.29
116197;1016;2003;45.95
116205;1019;2005;34.62
116212;1004;2001;76.19
116218;1017;2004;80.95
116218;1003;2002;12.50
116222;1004;2005;53.85
116226;1027;2003;78.38
116229;1020;2004;59.52
116229;1018;2001;16.67
116229;1022;2001;4.76
116239;1017;2002;56.25
116241;1010;2005;23.08
116247;1014;2004;26.19
116250;1017;2002;81.25
116260;1011;2004;76.19
116270;1005;2001;54.76
116279;1029;2003;43.24
116284;1027;2002;71.88
116289;1011;2001;30.95
116290;1017;2003;21.62
116290;1015;2002;90.62
116292;1015;2002;34.38
116298;1011;2005;30.77
116300;1021;2001;7.14
116309;1028;2001;92.86
116311;1020;2005;7.69
116320;1003;2005;19.23
116326;1017;2002;46.88
116327;1023;2003;18.92
116335;1022;2003;75.68
116345;1009;2005;7.69
116348;1006;2001;50.00
116352;1017;2003;18.92
116352;1004;2004;23.81
116357;1009;2001;16.67
116366;1026;2005;3.85
116371;1030;2001;59.52
116373;1008;2003;29.73
116376;1011;2005;34.62
116384;1006;2002;3.12
116391;1009;2002;25.00
116397;1010;2005;3.85
116406;1018;2004;76.19
116408;1027;2003;5.41
116418;1013;2002;37.50
116423;1025;2004;38.10
116428;1002;2003;0.00
116431;1008;2004;61.90
116441;1026;2002;71.88
116443;1018;2005;53.85
116453;1005;2001;92.86
116455;1010;2003;0.00
116458;1012;2005;53.85
116463;1002;2003;2.70
116464;1024;2005;38.46
116470;1027;2001;80.95
116471;1006;2003;32.43
116472;1027;2002;6.25
116474;1010;2001;76.19
116478;1021;2001;2.38
116488;1028;2002;100.00
116493;1008;2005;42.31
116501;1028;2003;2.70
116503;1012;2003;97.30
116512;1028;2004;0.00
116512;1030;2005;3.85
116522;1006;2001;0.00
116522;1014;2001;33.33
116532;1030;2002;37.50
116536;1013;2002;9.38
116546;1008;2003;29.73
116556;1006;2003;54.05
116560;1028;2001;57.14
116560;1003;2003;48.65
116566;1027;2005;53.85
116569;1021;2005;3.85
116579;1014;2005;23.08
116585;1002;2001;14.29
116587;1020;2005;26.92
116594;1016;2003;13.51
116604;1017;2004;4.76
116605;1021;2005;0.00
116615;1017;2004;19.05
116624;1015;2005;11.54
116627;1026;2002;31.25
116636;1015;2001;21.43
116639;1001;2002;53.12
116646;1014;2003;67.57
116651;1020;2001;7.14
116660;1004;2004;54.76
116667;1007;2003;81.08
116669;1015;2002;50.00
116678;1013;2003;2.70
116687;1008;2003;51.35
116696;1005;2002;15.62
116696;1030;2002;65.62
116697;1002;2003;0.00
116706;1015;2005;30.77
116711;1011;2002;0.00
116712;1028;2004;16.67
116717;1009;2001;66.67
116725;1017;2005;19.23
116731;1027;2002;3.12
116734;1026;2005;80.77
116736;1028;2002;40.62
116744;1018;2003;5.41
116752;1008;2004;26.19
116757;1010;2001;28.57
116762;1027;2003;2.70
116764;1001;2003;24.32
116771;1005;2001;4.76
116776;1012;2002;56.25
116776;1030;2004;42.86
116780;1030;2003;27.03
116784;1023;2005;26.92
116794;1011;2005;46.15
116801;1017;2001;0.00
116802;1002;2003;40.54
116803;1010;2003;94.59
116805;1011;2001;80.95
116807;1005;2003;48.65
116810;1017;2001;54.76
116817;1007;2003;40.54
116826;1007;2002;81.25
116836;1009;2005;38.46
116846;1024;2003;78.38
116850;1021;2004;35.71
116858;1021;2002;0.00
116859;1012;2001;33.33
116863;1012;2005;15.38
116871;1019;2003;29.73
116875;1024;2001;21.43
116882;1020;2003;75.68
116885;1013;2002;21.88
116887;1007;2002;43.75
116890;1025;2005;69.23
116891;1010;2002;31.25
116894;1016;2005;15.38
116898;1005;2002;18.75
116898;1023;2003;40.54
116905;1011;2005;7.69
