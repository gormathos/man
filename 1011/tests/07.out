1001: Excellent [75.00%]
1002: Very Good [61.45%]
1003: Very Good [70.92%]
1004: Very Good [69.44%]
1005: Excellent [79.60%]
1006: Very Good [64.29%]
1007: Very Good [64.84%]
1008: Excellent [78.13%]
1009: Very Good [73.57%]
1010: Excellent [83.39%]
1011: Excellent [74.83%]
1012: Excellent [76.22%]
1013: Excellent [83.10%]
1014: Very Good [70.43%]
1015: Very Good [67.92%]
1016: Excellent [82.40%]
1017: Excellent [79.24%]
1018: Excellent [85.66%]
1019: Excellent [78.89%]
1020: Very Good [71.45%]
1021: Very Good [73.56%]
1022: Excellent [81.09%]
1023: Excellent [85.53%]
1024: Excellent [80.99%]
1025: Very Good [71.04%]
1026: Very Good [65.09%]
1027: Excellent [83.93%]
1028: Excellent [85.64%]
1029: Excellent [82.38%]
1030: Excellent [76.55%]
